compiler/testData/codegen/boxInline/defaultValues/lambdaInlining/callableReferences/boundInlineClassMethod.kt

```diff
@@ -0,0 +1,17 @@
+// SKIP_INLINE_CHECK_IN: inlineFun$default
+// IGNORE_BACKEND: JVM
+// IGNORE_BACKEND_MULTI_MODULE: JVM, JVM_MULTI_MODULE_IR_AGAINST_OLD
+// FILE: 1.kt
+package test
+
+inline class C(val x: String) {
```

Comment:
Please add tests with the following:
(1) `inline class C(val x: Any?)`
(2) `inline class C(val x: Int)`

Reply:
done